src/components/icons/InfoIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+/**
+ * Remove SVG icon
```

Comment:
Why does this comment say "Remove SVG icon"?

Reply:
> Why does this comment say "Remove SVG icon"?

Sorry, that needs to be change. I will update it